### DISCUSSION TITLE
[Feature] Continue inference while switching between chats.

### DISCUSSION THREAD

**Discussion by imkebe** [open-webui/open-webui#1727] [General]
While locally hosted LLM's are not as fast as cloud ones, and there is an option to provide multiple independent hosting endpoints I would like to have two or more (might be able to configure it) active chats. Currently while I send a query I need to stay in the context of the conversation to finish it. When I switch to another one i lost the current context of the ongoing one. When it finishes it often mess up the conversation. There should be an pending icon in the left chat list if the query was send and the inference is ongoing.

**Comment by yz1797203443** [comment]
I also encountered the same problem. If you switch to another chat box before the model is answered, you will lose the chat content before switching
